### PR TITLE
Update Exchange-Teams-interact.md

### PR DESCRIPTION
**Pull request by AndreasExner**
Scheduling a meeting with Outlook and the Teams add-in for the user itself is supported without EWS/OAuth etc. When it comes to delegation it's not possible to schedule a meeting on behalf without some basic requirements. The lack of information in our public docs leads into widely confusion in our customers. Espacially as a result of several C-19 Teams rollouts. Please add a hint here. THX

https://dev.azure.com/Supportability/UC/_wiki/wikis/UC.wiki/3769/Teams-Delegate-Meeting-Requirements-and-Limitations

### CODE REVIEW

**Comment by scanum** [comment]
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/4523

**Comment by dstrome** [comment]
@carolynblanding - since you recently updated the topic for oauth, can you take a look at this change and confirm its accuracy?

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/Exchange-Teams-interact.md [hunk 1]
```diff
@@ -73,6 +73,9 @@ Microsoft Teams works with several Office 365 services to provide users with a r
 
 - To let Microsoft Teams work with Exchange on-premises, you must configure the new Exchange OAuth authentication protocol as described in [Configure OAuth authentication between Exchange and Exchange Online organizations](https://docs.microsoft.com/exchange/configure-oauth-authentication-between-exchange-and-exchange-online-organizations-exchange-2013-help).
 
+> [!NOTE]
+>The Outlook Teams add-in can be used to Schedule a Teams meeting for mailboxes hosted in Exchange on-premises. However, scheduling a Teams meeting on behalf of another user with Exchange on-premises requires Exchange 2013 CU9 and above and the new Exchange OAuth authentication protocol. Both delegate and delegator must have mailbox on Exchange on-premises.
```
Comment (JohanFreelancer9) [inline Teams/Exchange-Teams-interact.md:77]:
```suggestion
> The Outlook Teams add-in can be used to schedule a Teams meeting for mailboxes hosted in Exchange on-premises. However, scheduling a Teams meeting on behalf of another user with Exchange on-premises requires Exchange 2013 CU9 and above and the new Exchange OAuth authentication protocol. Both delegate and delegator must have a mailbox on Exchange on-premises.
```

File: Teams/Exchange-Teams-interact.md [hunk 2]
```diff
@@ -73,6 +73,9 @@ Microsoft Teams works with several Office 365 services to provide users with a r
...
+
 > [!NOTE]
 >For Exchange On-Premises and Teams integration, the required license needs to be assigned for the AAD synced user.
```
Comment (JohanFreelancer9) [inline Teams/Exchange-Teams-interact.md:80]:
```suggestion
> For Exchange On-Premises and Teams integration, the required license needs to be assigned for the AAD synced user.
```

**Comment by scanum** [comment]
@dstrome @carolynblanding PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by dstrome** [comment]
@AndreasExner thanks for the update. It looks like you're from Microsoft so please take a moment and link your github and corpnet accounts. When you do this we can more easily identify you as a Microsoft contributor. Thanks!

https://review.docs.microsoft.com/en-us/office-authoring-guide/contribute-get-started-setup-github-ro?branch=master#link-your-github-and-microsoft-accounts

**Pull request merged**